src/window.rs

```diff
@@ -732,6 +745,26 @@ impl Window {
         self.window.is_resizable()
     }
 
+    /// Sets the enabled window buttons.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Wayland / X11:** Not implemented.
+    /// - **Web / iOS / Android:** Unsupported.
+    pub fn set_window_buttons(&self, buttons: WindowButtons) {
```

Comment:
Naming idea: `[set_]enabled_buttons` or `[set_]visible_buttons` (`window` is at least redundant here)

Reply:
`[set_]enabled_buttons` sounds good, thanks.